### PR TITLE
hbinput: do not use private variables

### PR DESCRIPTION
**Pull request by m4rc1e**
Fixes https://github.com/fonttools/fonttools/issues/2819

### CODE REVIEW

File: Lib/diffenator/hbinput.py [hunk 1]
```diff
@@ -36,10 +36,11 @@ def __init__(self, font):
         glyph_set = self.font.ttfont.getGlyphSet()
         for name in glyph_set.keys():
             glyph = glyph_set[name]
+            glyf = glyph_set.glyfTable[name]
```
Comment (anthrotype) [inline Lib/diffenator/hbinput.py:39]: that'd only work for truetype, not for CFF, though..

Comment (anthrotype) [reply Lib/diffenator/hbinput.py:39]: .. but also the previous code didn't work for CFF anyway

Comment (m4rc1e) [reply Lib/diffenator/hbinput.py:39]: Thanks. My solution also only works for the latest so I'm going to redo this.

Comment (m4rc1e) [reply Lib/diffenator/hbinput.py:39]: yeah. Just reread the readme. Clearly states TTF. so I'll press ahead and just worry about glyf.

Comment (anthrotype) [reply Lib/diffenator/hbinput.py:39]: if you want the glyf table just get the glyf tabke fromn the font, as usual, don't get it from the glyph set object.
For CFF, you need to compute the control bounding box, charstrings have a `calcBounds` method that returns a 4-tuple

Comment (m4rc1e) [reply Lib/diffenator/hbinput.py]: yep, I'm now doing just `font['glyf'][name]` thanks.

Comment (m4rc1e) [reply Lib/diffenator/hbinput.py]: For diffenator2 we don't do any of this nonsense. Will be able to handle otf/ttf just fine. Just fixing this our CIs can work till diffenator2 is ready.

**Pull request merged**